Gems/WarehouseAutomation/Code/Source/WarehouseAutomationModuleInterface.cpp

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+#include "WarehouseAutomationModuleInterface.h"
+#include <AzCore/Memory/Memory.h>
+#include <WarehouseAutomationSystemComponent.h>
+#include <ConveyorBelt/ConveyorBeltComponent.h>
+#include <ProximitySensor/ProximitySensor.h>
+#include <WarehouseAutomation/WarehouseAutomationTypeIds.h>
+
+
+namespace WarehouseAutomation
+{
+    AZ_TYPE_INFO_WITH_NAME_IMPL(WarehouseAutomationModuleInterface,
+        "WarehouseAutomationModuleInterface", WarehouseAutomationModuleInterfaceTypeId);
+    AZ_RTTI_NO_TYPE_INFO_IMPL(WarehouseAutomationModuleInterface, AZ::Module);
+    AZ_CLASS_ALLOCATOR_IMPL(WarehouseAutomationModuleInterface, AZ::SystemAllocator);
+
+    WarehouseAutomationModuleInterface::WarehouseAutomationModuleInterface()
+    {
+        // Push results of [MyComponent]::CreateDescriptor() into m_descriptors here.
```

Comment:
Check comments such as this, which should not be left from the generated Gem

Reply:
In my opinion they are useful for clarification (future components will also need to be added here).
Should they be removed from the ROS2 gem as well then?

Reply:
Yes, these are not needed.